From a5f9005b3247f09f0a94e7bd497c68f6435668c0 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Sun, 19 Nov 2017 14:19:48 +0000
Subject: [PATCH 1/1] linux: use python3 by default

In Solus streamlink is built using python3 but python2 is the default.

Signed-off-by: Joey Riches <josephriches@gmail.com>
---
 src/config/streaming.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/config/streaming.json b/src/config/streaming.json
index b542b0fa..ee046fea 100644
--- a/src/config/streaming.json
+++ b/src/config/streaming.json
@@ -31,7 +31,7 @@
 			"exec": {
 				"win32": "pythonw.exe",
 				"darwin": "python",
-				"linux": "python"
+				"linux": "python3"
 			},
 			"fallback": {
 				"win32": [
-- 
2.15.0

